common/anomalies/hfe_anomaly.txt

```diff
@@ -226,7 +230,7 @@ hfe_anomaly_blight = {
 			}
 		}
 		modifier = {
-			factor = 0.1
+			factor = 0.2
```

Comment:
I think its to high.
It intended to be around 1 to 1 chance for player vs all AIs.

Reply:
Okay, if that is what you wish.

Reply:
Okay, done